docs/source/methods/energy_deposition.rst

```diff
@@ -0,0 +1,134 @@
+.. _methods_heating:
+
+=============================
+Heating and Energy Deposition
+=============================
+
+As particles traverse a problem, some portion of their energy is deposited at 
+collision sites. There are a variety of mechanisms that contribute to the
+energy deposition, including down-scattering from higher to lower energies,
+fission events, and reactions with positive Q-values. The information describing
+how much energy is deposited for a specific reaction is referred to as 
+"heating numbers" and can be computed using a program like NJOY with the
+``heatr`` module.
+
+These heating numbers are the product of reaction-specific coefficients and
+a reaction cross section
+
+.. math::
+
+    H(E) = \phi(E)\sum_i\rho_i\sum_rk_{i, r}
+
+and has units energy per time, typically eV / s.
+Here, :math:`k_{i, r}` are the KERMA [Kinetic Energy Release in Materials]
+coefficients for reaction :math:`r` of isotope :math:`i`.
+The KERMA coefficients have units energy :math:`\times` cross-section, e.g.
+eV-barn, and can be used much like a reaction cross section for the purpose
+of tallying energy deposition.
+
+KERMA coefficients can be computed using the energy-balance method with
+a nuclear data processing code like NJOY, which performs the following
+iteration over all reactions :math:`r` for all isotopes :math:`i`
+requested
+
+.. math::
+
+    k_{i, r}(E) = \left(E + Q_{i, r} - \bar{E}_{i, r, n} - \bar{E}_{i, r, \gamma}\right)\sigma_{i, r}(E)
+
+removing the energy of secondary neutrons and photons from the sum of
```

Comment:
removing the energy of neutral particles (neutrons and photons) that are transported away from the reaction site